src/modules/personenkontext/persistence/personenkontext.scope.ts

```diff
@@ -23,7 +23,6 @@ export class PersonenkontextScope extends ScopeBase<PersonenkontextEntity> {
             {
                 personId: findProps.personId,
                 referrer: findProps.referrer,
-                rolle: findProps.rolle,
                 personenstatus: findProps.personenstatus,
                 sichtfreigabe: findProps.sichtfreigabe,
```

Comment:
If you're removing this here, you should also remove it form the `FindProps`

Reply:
Done